.github/workflows/nightly.yml

```diff
@@ -198,7 +205,14 @@ jobs:
             ${{ matrix.run_tests_parameters }}
             -d zend_extension=opcache.so
             -d opcache.enable_cli=1
-            -d opcache.jit_buffer_size=16M
+            -d opcache.jit_buffer_size=64M
```

Comment:
We should probably move this to a common place, e.g. directly in `.github/actions/test-linux/action.yml` and enable it via parameter.

Reply:
Actually, all JIT parameters can directly be moved unconditionally in ` .github/actions/test-linux/action.yml`, and here we can just provide the `-d zend_extension=opcache.so`